### PR TITLE
chore(eve): update eve to 0.6.3

### PR DESCRIPTION
**Pull request by petrjasek**
it raises now `OriginalChanged` error when there is no update done in db,
which could be the case with system update (other updates change etag).

this case is now handled in eve backend layer.

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: superdesk/tests/__init__.py [hunk 1]
```diff
@@ -195,6 +203,9 @@ def setUp(self):
     def tearDown(self):
         if hasattr(self, 'ctx'):
             self.ctx.pop()
+        with self.app.app_context():
+            mongoclient = self.app.data.mongo.pymongo().cx
```
Comment (mugurrus) [inline superdesk/tests/__init__.py]: These 2 lines can be compacted into a single line

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**